packages/webview_flutter/webview_flutter_wkwebview/example/integration_test/webview_flutter_test.dart

```diff
@@ -845,7 +845,8 @@ void main() {
   });
 
   group('NavigationDelegate', () {
-    final String blankPage = "<!DOCTYPE html><head></head><body></body></html>";
+    const String blankPage =
+        '<!DOCTYPE html><head></head><body></body></html>;';
```

Comment:
You added a character to the end of the string here

Reply:
Oops that was indeed not my intention. Fixed.